recipes/thorvg/all/conanfile.py

```diff
@@ -54,7 +56,8 @@ class ThorvgConan(ConanFile):
         "loaders": "Enable File Loaders in thorvg",
```

Comment:
The option names in `options_description` should be adjusted to match their names in `options` and `default_options`.